src/frontend/shared/sidebar.vue

```diff
@@ -210,6 +213,11 @@ const classList = (path: string, withGap: boolean = false) => {
     'flex items-center gap-x-3': withGap,
   };
 };
+
+const extractLocaleFromLink = (link: string): string => {
+  const match = link.match(/^\/([a-z]{2})\//);
+  return match ? match[1] : 'en';
```

Comment:
Maybe a better default would be to show no locale:

```suggestion
  return match ? match[1] : '';
```

Reply:
Done.